pep-9999.rst

```diff
@@ -175,8 +190,33 @@ performed):
                 pep508 += " @ " + vcs + "+" requirement[vcs]
                 if "revision" in requirement:
                     pep508 += "@" + revision
+        markers = ""
+        for variable in (
+            "os_name",
+            "sys_platform",
+            "platform_machine",
+            "platform_python_implementation",
+            "platform_release",
+            "platform_system",
+            "platform_version",
+            "python_version",
+            "python_full_version",
+            "implementation_name",
+            "implementation_version",
+            "extra",
```

Comment:
Should we mention the ambiguity between `extra` and `extras`?

Reply:
That, or use `if-` prefix

Reply:
I added a note on the `extra` key, but I'd still like feedback on the `if-` prefix (or any other prefix/suffix/transformation)

Reply:
Yeah. I kinda like the `if` idea. At the same time if the verbosity will make it more unapproachable.

Reply:
@EpicWink after thinking about this a while, I realised that we should really be considering using a different name for the "enabled extras" instead. This way we preserve the use of "extra" (a known marker) for its intended purposes. And use a more explicit representation for extras we activate in the dependeny.

```toml
aiohttp = { version = "== 3.6.0", enabled-extras = ["speedups"], extra = "http" }
```

Just adding some food for thought here.

Reply:
Maybe. I'd use the prefix `select`. However, that is more to type for a much more common use-case.